ts/Dashboards/EditMode/Toolbar/CellEditToolbar.ts

```diff
@@ -198,15 +198,20 @@ class CellEditToolbar extends EditToolbar {
             );
             const x = cellOffsets.right - toolbarWidth - toolbarMargin;
             const y = cellOffsets.top + toolbarMargin;
-            const hiddenItems = cell.options.editMode?.hiddenToolbarItems || [];
 
-            // Temp - activate all items.
             objectEach(toolbar.menu.items, (item): void => {
-                if (hiddenItems.includes(item.options.id)) {
-                    item.deactivate();
-                } else {
+                if (!cell.options?.editMode?.toolbarItems) {
                     item.activate();
+                    return;
                 }
+
+                /* eslint-disable-next-line max-len */
+                if (cell.options.editMode.toolbarItems[item.options.id as keyof typeof cell.options.editMode.toolbarItems]?.enabled === false) {
```

Comment:
Can't we just extract the `cell.options.editMode.toolbarItems` to a short const here? :D